packages/playwright-test/src/reporters/base.ts

```diff
@@ -409,15 +422,14 @@ export function formatError(config: FullConfig, error: TestError, highlightCode:
       // Failed to read the source file - that's ok.
     }
   }
+
+  error.message = tokens.join('\n');
```

Comment:
I think we should also remove the name prefix `Error:` because by default message does not contain it.